### PR TITLE
Daemon doesn't fail if it can't init a reader-mode machine

### PR DESCRIPTION
**Pull request by jameshaydon**

### CODE REVIEW

**Comment by jkarni** [comment]
Should we build images for the daemon now, and try getting the tests to work with that instead? I imagine we'd want to be deploying daemons quite often.
(cc @geigerzaehler )

File: exe/Daemon.hs [hunk 1]
```diff
@@ -515,31 +527,32 @@ writeInputs id is nonce = do
...
+                  in if more' > 0
+                     then (True, HighFreq more')
+                     else (False, LowFreq)
+                -- Low frequency polling is every 10 seconds.
+                LowFreq -> (delta > lowFreqPollPeriod, LowFreq)
+        when shouldPoll $ do
+          logInfo Debug "Polling.." [("machine-id", getMachineId machineId)]
+          refreshAsReader machineId >> pure ()
+        modifyMachine machineId $ \m' -> pure (m' { machinePolling = newPoll }, ())
+      _ -> pure () -- Uninitialised readers are (for the moment) ignored.
```
Comment (jkarni) [inline exe/Daemon.hs]: Why not (low-freq) poll them as well?

File: cloudbuild.yaml [hunk 2]
```diff
@@ -202,4 +163,3 @@ steps:
       stack exec -- radicle-daemon &
 
       stack test :e2e
-      stack exec -- radicle test/machine-backends.rad radicle-server
```
Comment (geigerzaehler) [inline cloudbuild.yaml]: Please also remove the corresponding file `test/machine-backends.rad`. Otherwise we’ll forget about it

**Comment by geigerzaehler** [comment]
> Should we build images for the daemon now, and try getting the tests to work with that instead? I imagine we'd want to be deploying daemons quite often.

Seems likely that we want to build an image for the daemon in the future. But right now I see no immediate advantage to building and running it with docker.

File: cloudbuild.yaml (hunk 2 above)
Comment (jameshaydon) [reply cloudbuild.yaml]: :+1:

File: exe/Daemon.hs (hunk 1 above)
Comment (jameshaydon) [reply exe/Daemon.hs]: :+1:

File: src/Radicle/Daemon/Common.hs [hunk 3]
```diff
@@ -26,18 +27,19 @@ data Polling = HighFreq Int64 | LowFreq
...
+    , machineState        :: Bindings (PrimFns Identity)
+    , machineLastIndex    :: Maybe MachineEntryIndex
+    , machineMode         :: ReaderOrWriter
+    , machineSubscription :: TopicSubscription
+    , machineLastUpdated  :: Time.SystemTime
+    , machinePolling      :: Polling
     } deriving (Generic)
 
-newtype Chains id idx subs = Chains { getChains :: CMap.CMap id (Chain id idx subs) }
+data CachedMachine = UninitialisedReader Time.SystemTime | Cached Machine
```
Comment (jkarni) [inline src/Radicle/Daemon/Common.hs]: Can you put a comment saying what this `SystemTime` is for/about?

Comment (jameshaydon) [reply src/Radicle/Daemon/Common.hs]: :+1:

**Pull request merged**